fe1-web/src/features/index.ts

```diff
@@ -22,7 +22,6 @@ export function configureFeatures() {
   // configure features
   const eventConfiguration = events.configure();
   const walletConfiguration = wallet.configure();
-  const digitalCashConfiguration = digitalCash.configure();
 
   const notificationConfiguration = notification.configure();
```

Comment:
Is there a reason to have kept the configuration of the notification?

Reply:
I removed only the necessary code and the one that was not used anymore. digitalCashConfiguration is still used even id it is not necessary

Reply:
Ah I see, `digitalCashConfiguration` was not used anywhere else 👍. I might consider leaving `digitalCash.configure();` without assigning it to a variable so that the front-end doesn't break if unknown messages are received but this is up to you.